tests/openfl/utilities/test_verifiable_dataset.py

```diff
@@ -0,0 +1,263 @@
+import json
+from openfl.utilities.verifdataset.local_data_source import LocalDataSource
+from openfl.utilities.verifdataset.verifiable_dataset import DatasetFormat, VerifiableDataset
+import pytest
+import os
+import shutil
+from pathlib import Path
+from typing import List, Tuple
+
+@pytest.fixture(scope="class")
```

Comment:
can you use pyfakefs in your fixture ? this will prevent creating real files and folders, which is never good in unit tests. This is in the same vein as mock_aws.

Reply:
done